docker/shadow-hand/kinetic/base/Dockerfile

```diff
@@ -1,3 +1,9 @@
+#
+# Kinetic with build tools Dockerfile
```

Comment:
uninformative description. Please add more information about this image. It is better to use *LABEL Description="* I guess